src/overlays/actors/ovl_Door_Warp1/z_door_warp1.c

```diff
@@ -70,19 +70,27 @@ void DoorWarp1_SetupAction(DoorWarp1* this, DoorWarp1ActionFunc actionFunc) {
     this->actionFunc = actionFunc;
 }
 
-s32 func_808B849C(DoorWarp1* this, PlayState* play) {
-    s32 ret = 0;
+/**
+ * Returns the number of which remains are obtained inside this warp.
```

Comment:
This phrasing makes me think the function returns how many remains the player got in the warp.

What about something like this instead?
```suggestion
 * Returns a number representing which remains was obtained inside this warp.
```